io-engine/src/bdev/nvmx/channel.rs

```diff
@@ -329,12 +349,36 @@ impl IoQpair {
             (*connect_arg).poller = Some(poller);
         }
 
-        let r = receiver
-            .await
-            .expect("I/O qpair connection sender disappeared")
-            .map_err(|e| CoreError::OpenBdev {
-                source: e,
-            });
+        let r = tokio::select! {
```

Comment:
interesting, does tokio select fully work within our reactor? 
We use futures::select! in the rest of our code, might be worth considering a change if you're not sure.

Reply:
I saw no issues during testing. But I can replace it with, futures::select!, no problem here.

Reply:
Ah interesting, ok that's fine then :+1: